src/api/Feed.js

```diff
@@ -113,6 +113,23 @@ class Feed extends Base {
         this.taskLinksAPI = [];
     }
 
+    addFeedItem(id: string, item: FeedItem) {
+        if (!id || !item) {
+            return;
+        }
+
+        const cachedItems = this.getCachedItems(id);
+        const feedItems = cachedItems ? cachedItems.items : [];
+
+        if (feedItems.some(feedItem => feedItem.id === item.id)) {
+            return;
+        }
+
+        const updatedItems = [...feedItems, item].sort((a, b) => Date.parse(a.created_at) - Date.parse(b.created_at));
```

Comment:
Does this sort logic live elsewhere? If so, should be shared rather than copied?